hca-to-scea/script_spreadsheets/SRP151260/E-HCAD-44.idf.txt

```diff
@@ -0,0 +1,35 @@
+
```

Comment:
What is this `scripts_scpreadsheets` directory?
Which script uses it?
We already have a directory for the test inputs and the expected outputs.
The directory of files generated during the test run should not be added to git. If it is, delete it and add an entry to gitignore.